lntemp/node/config.go

```diff
@@ -219,6 +224,11 @@ func (cfg *BaseNodeConfig) GenArgs() []string {
 	case lntest.BackendPostgres:
 		args = append(args, "--db.backend=postgres")
 		args = append(args, "--db.postgres.dsn="+cfg.PostgresDsn)
+
+	case lntest.BackendSqlite:
+		args = append(args, "--db.backend=sqlite")
+		args = append(args, fmt.Sprintf("--db.sqlite.busytimeout=%ds",
```

Comment:
I think you can just do `%v` with `lntest.SqliteBusyTimeout` which will then render to `10s` automatically.